From f85a28baf1e79c7c9678ab676471ffe976a9ce26 Mon Sep 17 00:00:00 2001
From: hongjiasheng <118318@nd.com>
Date: Tue, 11 May 2021 20:04:46 +0800
Subject: [PATCH] patch_before

---
 patch.txt | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 patch.txt

diff --git a/patch.txt b/patch.txt
new file mode 100644
index 0000000..a2793f5
--- /dev/null
+++ b/patch.txt
@@ -0,0 +1,2 @@
+before patch.  
+
-- 
2.17.1.windows.2

